upgrade/UPGRADE-v8.1.0-dev.md

```diff
@@ -371,6 +371,18 @@ There you can find links to upgrade notes for other versions too.
         +        Assert::assertNotSame($errorIdFirstAccess, $errorIdSecondAccess);
         +    }
         ```
+- autowire service `ImageFacade` in data object factories ([#1476](https://github.com/shopsys/shopsys/pull/1476))
+    - `src/Shopsys/ShopBundle/Model/Payment/PaymentDataFactory.php`
+    ```diff
```

Comment:
Can you please add to this diff the whole change of the constructor?
```diff
    public function __construct(
        PaymentFacade $paymentFacade,
        VatFacade $vatFacade,
-       Domain $domain
+       Domain $domain,
+       ImageFacade $imageFacade
    ) {
-       parent::__construct($paymentFacade, $vatFacade, $domain);
+       parent::__construct($paymentFacade, $vatFacade, $domain, $imageFacade);
```

Reply:
Thx. fixed